tests/install/openqa_webui.pm

```diff
@@ -56,6 +57,7 @@ git clone https://github.com/os-autoinst/openQA.git
 cd openQA
 for p in $(cpanfile-dump); do echo -n "perl($p) "; done | xargs zypper --non-interactive in -C
 cpanm -nq --installdeps .
+tools/generate-packed-assets || (sleep 1; tools/generate-packed-assets || exit 1) || (sleep 1; tools/generate-packed-assets || exit 1)
```

Comment:
I guess if our tests show that assets over CDN might not be always reachable then tools/generate-packed-assets should already apply the retry within itself.

Reply:
I didn't check if this script retries, but if not, we could move the functionality to tools/generate-packed-assets checking it